src/components/TweetCreationForm.tsx

```diff
@@ -106,6 +106,7 @@ const TweetCreationForm: React.FC<TweetCreationFormProps> = ({ repliedId }) => {
         <div className="flex-grow-1 ms-2">
           <Form.Control
             as="textarea"
+            style={{ resize: "none", paddingLeft: "8px" }}
```

Comment:
1) I haven't found how "resize" affects the textarea
2) Why you took 8px? you guessed
![image](https://user-images.githubusercontent.com/103836064/193134186-e541531b-ac56-47d6-9a09-a6ecf71df55e.png)

Reply:
resize - 
![image](https://user-images.githubusercontent.com/99827460/193135705-4c367afb-22fe-4809-8eed-8cbaecec963d.png)

Reply:
8px because i align this component to the bottom
![image](https://user-images.githubusercontent.com/99827460/193136813-d40e0b55-915d-4c46-a2b6-7e162090d894.png)